### PR TITLE
Hide tooltip when hovering over the attribution group and between top level groups.

### PR DESCRIPTION
**Pull request by stanislawosinski**

### CODE REVIEW

**Comment by valscion** [comment]
Thanks! Would you be able to showcase the difference in behavior with a small screencap or something? It would make it easier to understand how this behavior has changed now ☺️

**Comment by valscion** [comment]
Thanks! Would you be able to showcase the difference in behavior with a small screencap or something? It would make it easier to understand how this behavior has changed now ☺️

**Comment by stanislawosinski** [comment]
Before:

https://user-images.githubusercontent.com/628360/105147827-fb0d9b80-5b01-11eb-9c6a-f0241c86ed34.mp4

After:

https://user-images.githubusercontent.com/628360/105147856-0365d680-5b02-11eb-9713-0aa3e54a0b40.mp4

**Comment by valscion** [comment]
Nice, thank you! Looks great!

Would you be able to write a small note on the changelog about this?

File: client/components/Treemap.jsx [hunk 1]
```diff
@@ -105,9 +105,12 @@ export default class Treemap extends Component {
       },
       onGroupDoubleClick: preventDefault,
       onGroupHover(event) {
-        // Ignoring hovering on `FoamTree` branding group
-        if (event.group && event.group.attribution) {
+        // Ignoring hovering on `FoamTree` branding group and the root group
+        if (event.group && (event.group.attribution || event.group === this.get('dataObject'))) {
           event.preventDefault();
+          if (props.onMouseLeave) {
```
Comment (valscion) [inline client/components/Treemap.jsx:111]: Do we need this conditional? Is there a place in the code where `props.onMouseLeave` would not be specified?

Comment (stanislawosinski) [reply client/components/Treemap.jsx:111]: I made the condition analogous to the `props.onGroupHover` check below, which is also there but probably not required:

```
        if (props.onGroupHover) {
          props.onGroupHover.call(component, event);
        }
```

I can remove the `props.onMouseLeave` condition, not a problem.

**Review by th0r — APPROVED** [review]

**Comment by th0r** [comment]
Thanks!

**Pull request merged**